tests/commands/predict_test.py

```diff
@@ -158,3 +160,60 @@ def predict_json(self, inputs: JsonDict, cuda_device: int = -1) -> JsonDict:
             assert set(result.keys()) == {"span_start_logits", "span_end_logits",
                                           "span_start_probs", "span_end_probs", "best_span",
                                           "best_span_str", "overridden"}
+
+    def test_alternative_file_formats(self):
+        tempdir = tempfile.mkdtemp()
+        infile = os.path.join(tempdir, "inputs.txt")
+        outfile = os.path.join(tempdir, "outputs.txt")
+
+        @Predictor.register('bidaf-csv')  # pylint: disable=unused-variable
```

Comment:
What variable is it complaining about?  The class?  I'm surprised this works, if that's the case - I thought pylint scoping worked differently than this.

Reply:
it's complaining about the class I believe because we never instantiate it directly, only through `Predictor.from_params`

Reply:
Interesting - I thought that inline comments like this only applied to the current line, so I'm surprised this works.  Makes me wonder if all of our other inline annotations are wrong...

Reply:
surprisingly, pylint complains if you move the disable to the next line. I don't really understand this, but I guess it has something to do with using the `register` decorator